src/Game.java

```diff
@@ -0,0 +1,149 @@
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.*;
+
+public class Game {
+    private static final Random RANDOM = new Random();
+
+    private final static int PLAY = 1;
+    private final static int EXIT = 2;
+
+    Scanner scanner = new Scanner(System.in);
+
+    private static final int MAX_ERRORS = 8;
+    private String wordToFind;
+    private char[] wordFound;
+    private int nbErrors;
+    private ArrayList<String> letters = new ArrayList<>();
+    private ArrayList<String> lettersFromFile = new ArrayList<>();
+
+    public void hangman(){
+    boolean isException = true;
+        while (isException) {
+        try {
+            int option = -1;
+            while (option != EXIT) {
+
+                System.out.println("Wybierz opcję:");
+                System.out.println("1 - graj");
+                System.out.println("2 - koniec programu");
+                option = scanner.nextInt();
+                scanner.nextLine();
+
+                switch (option) {
+                    case EXIT:
+                        System.out.println("Bye bye!");
+                        scanner.close();
+                        break;
+                    case PLAY:
+                        play();
+                        break;
+                    default:
+                        System.out.println("Wybrałeś nieprawidłową opcję");
+                }
+            }
+            isException = false;
+        } catch (InputMismatchException e) {
+            System.out.println("Podałeś niepoprawny znak.");
+            scanner.nextLine();
+        }
+    }
+}
+
+    private String nextWordToFind() {
+        return Keywords.WORDS[RANDOM.nextInt(Keywords.WORDS.length)];
+    }
+
+    private void addWord(String word) {
```

Comment:
Same with this stuff. Loading words from file. I mean, you can create 2 classes that share interface (because they are pretty much doing same thing) one for words array and second one for loading words from a file